libraries/aws/provisioning/src/aws_iot_provisioning_serializer.c

```diff
@@ -346,6 +346,11 @@ AwsIotProvisioningError_t _AwsIotProvisioning_SerializeCreateKeysAndCertificateR
                                                                *pSerializationBuffer,
                                                                *pBufferSize );
 
+    if( status != AWS_IOT_PROVISIONING_SUCCESS )
```

Comment:
Make sure we have a task to eliminate these `goto`s.

Reply:
Yes, this is old code. I plan to create a hygiene epic for making all quality improvements I have documented here: https://quip-amazon.com/56ZDAgTB0Ln5